install_config/configuring_openstack.adoc

```diff
@@ -40,7 +39,55 @@ Consult your OpenStack administrators for values of the `*OS_*` variables, which
 are commonly used in OpenStack configuration.
 
 [[openstack-configuring-masters]]
-== Configuring Masters
+== Configuring {product-title} Masters for OpenStack
+
+You can set OpenStack configuration on your {product-title} masters and nodes two different ways:
```

Comment:
"You can set an OpenStack configuration on your {product-title} master and node hosts in two different ways:"